iOSClient/Data/NCElementsJSON.swift

```diff
@@ -34,6 +34,9 @@ import UIKit
     
     @objc public let capabilitiesFileSharingApiEnabled:         Array = ["ocs","data","capabilities","files_sharing","api_enabled"]
     @objc public let capabilitiesFileSharingPubPasswdEnforced:  Array = ["ocs","data","capabilities","files_sharing","public","password","enforced"]
+    @objc public let capabilitiesFileSharingDefaultPermissions  Array = ["ocs","data","capabilities","files_sharing","default_permissions"]
+    // NC >= 23
```

Comment:
Line 37 - ":"

Reply:
Good catch.. Made this in the GH editor without building 😅

Reply:
But where you use it ?